tensorrt_llm/_torch/visual_gen/executor.py

```
@@ -30,12 +36,222 @@
+                    break
+                process.start()
+        except BaseException as e:
+            self._spawn_error = e
+            logger.error(f"VisualGen worker process spawn failed: {e}")
+        finally:
+            self._spawn_complete.set()
+
+        while not self._release.is_set():
+            try:
+                self._wait_for_release()
+            except BaseException as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `BaseException`

(BLE001)